rfd/0109-cloud-agent-upgrades.md

```diff
@@ -327,6 +327,30 @@ to be changed to follow the same pattern of registering a release channel and in
 by ec2 discovery.
 
 
+### ECS Fargate Agents
+
+When a user uses the Teleport Discover flow to connect an RDS, a Teleport agent is automatically provisioned using ECS Fargate. We need a
+way to keep these agents updated. One idea is to run a separate goroutine in Teleport that periodically checks for an available update and
+uses the ECS Fargate API to update Teleport agents.
```

Comment:
As I understand Fargate instances can't upgrade themselves, that's why we have this separate problem, while other clients in this RFD are supposed to upgrade themselves? Maybe it's worth mentioning?

Reply:
I've added some more context. The auto updates functionality depends on systemd or Kubernetes, but these are not available for Fargate agents.